src/style.css

```diff
@@ -0,0 +1,129 @@
+/* #region reset */
+a,
+p {
+  margin: 0;
+  text-decoration: none;
+}
+/* #endregion */
+/* #region variables*/
+:root {
+  --img-gap: calc(50% - 160px / 2);
+  --padding-elements: 0 16px;
+}
+/* #endregion */
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__img {
+  margin: 32px var(--img-gap) 40px;
+  width: 160px;
+  height: 134px;
+  object-fit: cover;
+  object-position: center;
+  display: block;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  padding: var(--padding-elements);
```

Comment:
Please try to use padding for card for preventing reuse padding in every element

![image](https://user-images.githubusercontent.com/54936044/222401244-f37e27fb-db3f-4b00-a4d2-b7cfea066e8c.png)